src/Exception/RuntimeException.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
I think it would be nice to update all headers to point to the repository (as we have in many others repos already, for example see any expressive repo)